Mail/Views/Thread/Message/MessageView.swift

```diff
@@ -50,13 +52,20 @@ struct MessageView: View {
         threadForcedExpansion[message.uid] == .expanded
     }
 
+    init(threadForcedExpansion: Binding<[String: MessageExpansionType]>, message: Message, isSingleMessage: Bool) {
+        _threadForcedExpansion = threadForcedExpansion
+        self.message = message
+        self.isSingleMessage = isSingleMessage
+    }
+
     var body: some View {
         VStack(spacing: 0) {
             MessageHeaderView(
                 message: message,
                 isMessageExpanded: Binding(get: {
                     isMessageExpanded
                 }, set: { newValue in
+                    guard !isSingleMessage else { return }
```

Comment:
Maybe we can compute it on the fly ?
```suggestion
                    guard threadForcedExpansion.count > 1 else { return }
```